x/genutil/client/cli/migrate.go

```diff
@@ -28,9 +29,10 @@ const flagGenesisTime = "genesis-time"
 // Ref: https://github.com/cosmos/cosmos-sdk/issues/5041
 var migrationMap = types.MigrationMap{
 	"v0.36": v036.Migrate,
-	"v0.38": v038.Migrate, // NOTE: v0.37 and v0.38 are genesis compatible
+	"v0.38": v038.Migrate, // NOTE: v0.37 and v0.38 are genesis compatible.
 	"v0.39": v039.Migrate,
-	"v0.40": v040.Migrate,
+	"v0.40": v040.Migrate, // NOTE: v0.40, v0.41 and v0.42 are genesis compatible.
```

Comment:
When we migrate from v0.39 to v0.{40,41,42}, do you prefer to have:
1. `simd migrate v0.40 old_039_genesis.json`
2. `simd migrate v0.42 old_039_genesis.json`
?

both commands do the exact same thing, since their genesis are compatible. it's just a UX question.

Reply:
(2) I think?

Reply:
OK, i switched to (2). It's technically a breaking change though, I'll add a CL entry